Cargo.toml

```diff
@@ -60,6 +60,7 @@ defmt = ["dep:defmt", "heapless/defmt-03"]
 "proto-sixlowpan" = ["proto-ipv6"]
 "proto-sixlowpan-fragmentation" = ["proto-sixlowpan", "_proto-fragmentation"]
 "proto-dns" = []
+"proto-domainname" = []
```

Comment:
The feature's name could be confusing as "domainname protocol" sounds a lot like DNS. I suggest `proto-dhcpv4-domainname`.

Reply:
Good suggestion!